bin/commands/init/security/index.ts

```diff
@@ -77,5 +67,10 @@ export function execute(dryRun?: boolean, ignoreSecurityFailures?: boolean) {
   common.printMessage(`WARNING: Due to the limitation of the ZWEI${securityPrefix} job, exit with 0 does not mean`);
   common.printMessage(`         the job is fully successful. Please check the job log to determine`);
   common.printMessage(`         if there are any messages indicating a problem.`);
+  if (esmWarning) {
```

Comment:
I know this pre-dates the current PR, but I'm curious - why is this a warning instead of an error? Is there any reason why we shouldn't convert this to an error and quit out of the init command before JCL is submitted?

Reply:
There are couple things, which must be done together:
* When preparing `jcllib`, `ZWEGEN00` is deleting all JCLs, which are NOT for the current ESM
  * `ZWEGEN00` is traversing control blocks to get the used ESM
* This command then must use the current ESM
* Generally there is an approach to do the maximum - like if you set `infinispan` for caching, we skip `VSAM` and reporting success
* As far as I know, there is only one complicated task: for example: if you would like to see resolved TSS JCLs on LPAR with RACF or ACF2

Reply:
I'll open a new issue as a low priority. I am also OK with the current state since it yields basically the same user experience - if the wrong security manager is set, the JCL will run and return with an error. Thank you